sentry/src/main/java/io/sentry/jvmti/Frame.java

```diff
@@ -0,0 +1,86 @@
+package io.sentry.jvmti;
+
+import java.lang.reflect.Method;
+import java.util.Arrays;
+
+/**
+ * Class representing a single call frame.
+ */
+public final class Frame {
+    /**
+     * Java method this frame is for.
+     */
+    private final Method method;
```

Comment:
It doesn't seem like this is used other than in `toString` — was there a plan to use it somewhere else?